src/handlers/add-issue.ts

```diff
@@ -0,0 +1,29 @@
+import { Context } from "../types";
+import { IssuePayload } from "../types/payload";
+
+export async function addIssue(context: Context) {
+  const {
+    logger,
+    adapters: { supabase },
+  } = context;
+  const { payload } = context as { payload: IssuePayload };
+  const markdown = payload.issue.body + " " + payload.issue.title || "";
```

Comment:
```suggestion
  const markdown = payload.issue.body + " " + payload.issue.title || null;
```

Empty string seems like an impossibility. You can't post empty comments so this actually should probably throw an error.